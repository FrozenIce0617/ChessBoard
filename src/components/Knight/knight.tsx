import React from 'react';

class Knight extends React.Component {
  render() {
    return (
      <div
        style={{
          fontSize: '10vw',
          textAlign: 'center',
          lineHeight: '10vw',
          backgroundColor: 'rgba(90, 90, 90, 0.5)',
        }}
      >
        â™˜
      </div>
    );
  }
}

export default Knight;
