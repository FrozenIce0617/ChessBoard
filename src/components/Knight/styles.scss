.knightContainer {
  font-size: 10vw;
  text-align: center;
  line-height: 10vw;
  background-color: rgba(90, 90, 90, 0.5);
}
