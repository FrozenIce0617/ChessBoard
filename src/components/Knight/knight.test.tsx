import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Knight from './knight';

describe('Square Component', () => {
  it('render component', () => {
    const component = shallow(<Knight />);

    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
