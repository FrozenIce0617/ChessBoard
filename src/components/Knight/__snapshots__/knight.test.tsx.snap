// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square Component render component 1`] = `
<div
  className="knightContainer"
>
  â™˜
</div>
`;
